### ISSUE TITLE
Conflicting dependencies in 2.0.0-alpha30

### ISSUE THREAD

**Issue by metametadata** [metosin/compojure-api#415]
## Library Version
2.0.0-alpha30

## Problem
`lein deps :tree` produces:

```
Possibly confusing dependencies found:
[metosin/muuntaja "0.6.4"] -> [com.cognitect/transit-clj "0.8.313"] -> [com.cognitect/transit-java "0.8.337"] -> [commons-codec "1.10"]
 overrides
[metosin/ring-swagger "0.26.2" :exclusions [joda-time]] -> [metosin/ring-http-response "0.9.1"] -> [ring/ring-core "1.7.1"] -> [ring/ring-codec "1.1.1"] -> [commons-codec "1.11"]
 and
[metosin/ring-swagger "0.26.2" :exclusions [joda-time]] -> [ring/ring-core "1.7.1"] -> [ring/ring-codec "1.1.1"] -> [commons-codec "1.11"]
 and
[metosin/ring-http-response "0.9.1" :exclusions [joda-time]] -> [ring/ring-core "1.7.1"] -> [ring/ring-codec "1.1.1"] -> [commons-codec "1.11"]

Consider using these exclusions:
[metosin/ring-swagger "0.26.2" :exclusions [commons-codec joda-time]]
[metosin/ring-http-response "0.9.1" :exclusions [commons-codec joda-time]]
```

Possible solution to catch conflicts in the future automatically is to add `:pedantic? :abort` into `project.clj`.

**Comment by metametadata** [comment]
Will do.

**Comment by miikka** [comment]
Fixed in `2.0.0-alpha31`.
